features/works/chapter_edit.feature

```diff
@@ -435,3 +436,22 @@ Feature: Edit chapters
       And I press "Post Without Preview"
     Then I should see "Publication date can't be in the future."
     When I jump in our Delorean and return to the present
+
+
+  Scenario: You should be able to add a chapter with a co-creator who has an
```

Comment:
There are two sets of pending tests for AO3-4998 in this file, they should be enabled or removed.